feedi/sources/rss.py

```diff
@@ -14,6 +14,12 @@
 feedparser.USER_AGENT = USER_AGENT
 
 
+def strip_html(text):
+    """Remove html tags from a string"""
+    import re
+    clean = re.compile('<.*?>|<.*?$')
```

Comment:
I'd rather use some beautiful soup method for this instead of regexes

Reply:
eg. `BeautifulSoup(text, 'lxml').text` should work

Reply:
Doesn't seem to be parsing correctly. Still get `<p>` and similar in the output.

Reply:
> Doesn't seem to be parsing correctly. Still get <p> and similar in the output.

Make sure you are adding the `.text` bit:

```python
In [1]: from bs4 import BeautifulSoup

In [2]: BeautifulSoup('<p>this is html</p>', 'lxml').text
Out[2]: 'this is html'

In [3]: BeautifulSoup('this is NOT html', 'lxml').text
Out[3]: 'this is NOT html'
```

> I'd also like to avoid turning the whole thing into a soup, that's a lot of parsing just for a title. I'm concerned about the time needed to parse a post.

Are you positive that parsing the soup is slower than compiling a regex? I would't know, but even if it was the difference wouldn't be noticeable in the context of feed parsing, which implies making several http requests (and it's run in the background, so it's not really noticeable by the user either).

Using the soup not only is (from my perspective) more readable and more consistent with the rest of the code, it avoids possible weirdness that may result from truncating the html before parsing it.

Reply:
Might be due to the fact that I was passing in a partial description. I'll parse a full soup of the description in-function for now, but will look at another PR for refactoring to use the `lxml` module for speed later. BS4 does use the lxml parser, [but it's still slow](https://www.reddit.com/r/Python/comments/7dz0sf/regex_vs_bs4_for_html_parsing/).